thesis/economy.tex

```diff
@@ -0,0 +1,41 @@
+\chapter{Project economy}
+When deciding the price model of the product, the total costs of the deployed system and development tools should be compared to the consumers expectations.
+
+Racket prices are around kr. 1000 - 1499 and the most owned sport-gadgets was \Glsi{endomondo}, free with premium subscription of \$2.5 and \glsi{gopro} with price range of \$230 - \$480, chapter \ref{ch:userSurvey}.
+
+\section{Development}
+The goal of this project was to create the products with \gls{oss} and it should preferably be free.
+The project uses a single package that is a commercial product, see chapter \ref{ch:dependencies}.
+As this package could be substituted by other packages the economic foot print of the libraries and frameworks used for the project could become zero.
+
+\section{Production}
+There are costs to running the applications in production.
+The platform specific application stores requires developer licences that comes with a fee.
+These are \$25 one-time \citep{economy:play} and \$99 per year \citep{economy:appStore} for \textit{Google Play} and \textit{Apple AppStore} respectively.
+
+Several solutions are available for hosting the back-end.
+As it is a \glsi{node} application any provider either serving a \glsi{node} environment or a container run-time can be used.
+In table \ref{tbl:hosting} is a short overview of possible providers and their estimated prices.
+All the container providers are only billing actual traffic to the running instances.
+This means the actual costs of running the back-end can differ significantly from the listed prices according to the popularity of the product.
+
+Further more a \gls{dns} is necessary for the front-end applications to communicate with the back-end, ignorant of the actual location of the resources, i.e. the IP addresses.
```

Comment:
not really necessary... surely its more resillient because changing the server would not require a new app deploy.. maybe explain that?  Idunno .. just technically its not necessary ;)